src/slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1])
 
 # Output the second-to-last element: 9
-print()
+print(a[len(a) - 2])
```

Comment:
This is the way we might do it in javascript, but you can just use `a[-2]` to get the second to last index, similar to what you have on line 21.